DigitalLearningSolutions.Data.Tests/Services/SectionServiceTests.cs

```diff
@@ -98,9 +98,67 @@ public void GetSectionAndTutorialsBySectionIdForCustomisation_returns_null_if_se
             using (new AssertionScope())
             {
                 result.Should().BeNull();
-                A.CallTo(() => tutorialContentDataService.GetTutorialsBySectionId(A<int>._, 1))
+                A.CallTo(() => tutorialContentDataService.GetTutorialsBySectionIdAndCustomisationId(A<int>._, 1))
                     .MustNotHaveHappened();
             }
         }
+
+        [Test]
+        public void GetSectionsForApplication_calls_data_service()
+        {
+            // Given
+            var sectionOne = new Section(1, "Section");
+            var sectionTwo = new Section(2, "Second Section");
+            var sections = new List<Section> { sectionOne, sectionTwo };
+
+            var tutorial = new Tutorial(1, "Tutorial", true, true);
+            var tutorials = new List<Tutorial> { tutorial };
+
+            A.CallTo(
+                () => sectionContentDataService.GetSectionsForApplication(1)
+            ).Returns(sections);
+
+            A.CallTo(
+                () => tutorialContentDataService.GetTutorialsForSection(A<int>._)
+            ).Returns(tutorials);
+
+            // When
+            var result = sectionService.GetSectionsWithTutorialsForApplication(1);
+
+            // Then
+            A.CallTo(() => sectionContentDataService.GetSectionsForApplication(1))

```

Comment:
As there are two data services being called should we assert on both of them here rather than just the one?

Reply:
Right, added a check on the tutorialContentDataService and modified the method slightly so the test passes